.werft/build.ts

```diff
@@ -58,7 +58,7 @@ async function run(context: any) {
 
 
     await validateChanges(werft)
-    await prepare(werft)
+    config = await prepare(werft, config)
```

Comment:
I would very much like to keep the config a const and not mutate the config options anywhere ☺️ It makes it easier to debug the the script if you keep the configuration parsing early and treat it as immutable for the rest of the job. So if possible, please move `lookForBranchNameWithVM` to job-config.ts and invoke the function there instead ☺️